.gitignore

```diff
@@ -173,3 +173,6 @@ tools/prometheus*
 
 # ray project files
 project-id
+
+# Cython example files
+doc/examples/cython/**/*
```

Comment:
I don't think this should be ignored - is there a reason you added it?

Reply:
These files are newly generated after build ray on my pc, and they didn't add into the git system. so I added just for my own convenience.